gtsfm/multi_view_optimizer.py

```diff
@@ -46,14 +45,14 @@ def create_computation_graph(
         self,
         images_graph: List[Delayed],
         num_images: int,
-        keypoints_graph: List[Delayed],
+        keypoints_graph: List[Keypoints],
```

Comment:
Confirming that indeed the type is `Keypoints` and not `Delayed`, since the `CorrespondenceGenerator` landed.

https://github.com/borglab/gtsfm/blob/master/gtsfm/scene_optimizer.py#L247